modules/javafx.web/src/main/java/com/sun/javafx/webkit/drt/DumpRenderTree.java

```diff
@@ -323,6 +323,8 @@ private static native void didClearWindowObject(long pContext,
     private static native boolean dumpChildFramesAsText();
     private static native boolean dumpBackForwardList();
     protected static native boolean shouldStayOnPageAfterHandlingBeforeUnload();
+    protected static native String[] openPanelFiles();
+    protected static native String testURL();
```

Comment:
DRT already has the test url, DumpRenderTree::testPath is the one. No need to get it from native C++ TestRunner class.